sympy/sets/__init__.py

```diff
@@ -1,13 +1,14 @@
 from .sets import (Set, Interval, Union, FiniteSet, ProductSet,
         Intersection, imageset, Complement, SymmetricDifference,
         DisjointUnion)
+
 from .fancysets import ImageSet, Range, ComplexRegion
 from .contains import Contains
 from .conditionset import ConditionSet
 from .ordinals import Ordinal, OmegaPower, ord0
 from .powerset import PowerSet
 from ..core.singleton import S
-
+from .handlers.comparison import _eval_is_eq  # noqa:F401
```

Comment:
Why is this import needed?

Reply:
Because without it, the dispatcher did not find _eval_is_eq for the sets when they were in the handlers file